<?xml version="1.0"?>
<launch>
    <!-- Start a bag file to record data -->
        <node pkg="rosbag" type="record" name="data"
                args="-O $(find asl_fixedwing)/data/rompc.bag
                      /plane/position
                      /plane/velocity
                      /plane/euler
                      /plane/bodyrate
                      /plane/actuators
                      /rompc/pos_error
                      /rompc/vel_error
                      /rompc/att_error
                      /rompc/attrate_error
                      /rompc/ubar
                      /rompc/u
                      /rompc/zbar
                      /rompc/zhat"
    />

    <!-- Start ROMPC node 
    MODEL: gazbeo or skywalker
    CTRL_TYPE: body_rate or ctrl_surf
    TARGET_TYPE: sgf, slf, or stf
    MODEL_TYPE: standard or cfd
    -->
	<node pkg="asl_fixedwing" type="rompc_node" name="rompc_node" output="screen">
		<param name="T_RESET" value="2.0" />
		<param name="CTRL_RATE" value="100.0" />
        <param name="CTRL_PATH" value="$(find asl_fixedwing)/src/rompc/" />
        <param name="MODEL" value="gazebo" />
        <param name="CTRL_TYPE" value="body_rate" />
        <param name="TARGET_TYPE" value="slf" />
        <param name="MODEL_TYPE" value="standard" />
	</node>
</launch>
